### PR TITLE
docs(README): Add known issues for M365 ME debugging issue

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: templates/bot/js/m365/README.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ This template provides a simple functionality to search for `npm` packages and r
   
     ![InOutlook](https://user-images.githubusercontent.com/11220663/167869656-20225f14-f982-4e47-8dd0-050285d56853.png)
 
+## Known issue
+
+You might see `Something went wrong` from the message extension while local debugging this application. This is a known issue to us and we are actively working on a fix to resolve it. Meanwhile, to proceed with local debug, you can delete those three files: `config.local.json`, `local.userdata` and `state.local.json` and press `F5` again.
```
Comment (kuojianlu) [inline templates/bot/js/m365/README.md:68]: Use the file pah, like `.fx/configs/config.local.json`?

Comment (zhenjiao-ms) [inline templates/bot/js/m365/README.md]: Does it work if only delete state.local.json? I am thinking of asking people to do as little as possible

Comment (zhenjiao-ms) [inline templates/bot/js/m365/README.md]: Better to provide the path of the files

**Review by kuojianlu — DISMISSED** [review]

File: templates/bot/js/m365/README.md (hunk 1 above)
Comment (MuyangAmigo) [reply templates/bot/js/m365/README.md]: Deleting all of them seems to be cleaner

Comment (MuyangAmigo) [reply templates/bot/js/m365/README.md]: Updated.

**Review by kuojianlu — APPROVED** [review]

**Review by zhenjiao-ms — APPROVED** [review]

**Pull request merged**